### ISSUE TITLE
Ошибки проверки запросов на соответствие стандартам отображать в редакторе запроса.

### ISSUE THREAD

**Issue by DmitryShvaika** [1C-Company/1c-edt-issues#1400]
### Описание проблемы

Сейчас ошибки проверки запросов отображаются  тексте редактора модуля и в панели ошибок. Однако при вводе нового (или редактировании существующего) запроса через редактор запроса эти ошибки не отображаются.

![image](https://github.com/1C-Company/1c-edt-issues/assets/54451084/9eab036d-b93c-4a83-8f18-ff855a816841)

### Описание решения проблемы

Предлагаю отображать ошибки о нарушении стандартов в редакторе запросов непосредственно

**Comment by adminimusRU** [comment]
Уже давно в редакторе отображаются, даже справа были полоски у полосы прокрутки. 
Версия старая? Или темная тема шалит?

**Comment by DmitryShvaika** [comment]
> Уже давно в редакторе отображаются, даже справа были полоски у полосы прокрутки. Версия старая? Или темная тема шалит?

Ни то, ни другое. Версия последняя Ruby 2023.3.4. И другие ошибки редактор нормально отображает. А у вас этот код отображает ошибки?
```
// @strict-types - директива для edt - проверка модуля в строгих типах

#Область ПрограммныйИнтерфейс

// Тест.
// 
// Возвращаемое значение:
//  Строка
//
Функция Тест() Экспорт

	Возврат "ВЫБРАТЬ
			|	1 КАК Поле1
			|ПОМЕСТИТЬ Таб
			|ИЗ
			|	(ВЫБРАТЬ
			|		1) КАК ВложенныйЗапрос1
			|		ЛЕВОЕ СОЕДИНЕНИЕ (ВЫБРАТЬ
			|			1) КАК ВложенныйЗапрос2
			|		ПО ИСТИНА
			|"

КонецФункции

#КонецОбласти

```

**Comment by adminimusRU** [comment]
Да, действительно именно проверки из плагина в редакторе не видно.
